cli/azd/internal/repository/detect_confirm.go

```diff
@@ -168,7 +168,7 @@ func (d *detectConfirm) render(ctx context.Context) error {
 			status = " " + output.WithSuccessFormat("[Added]")
 		}
 
-		d.console.Message(ctx, "  "+color.BlueString(projectDisplayName(svc))+status)
```

Comment:
Again - not sure if this was an intentional choice to use colors directly vs using our output wrappers.
@weikanglim

Reply:
It was intentional, see [previous response](https://github.com/Azure/azure-dev/pull/2997/files/f7ff8df32dd022440052958553829ebe05508ff1#r1396396529).

Reply:
Rolling back these color changes for now but we should be consistent and intentional and introduce new color wrappers if needed to take the guess work on the correct color to use.